kong/reports.lua

```diff
@@ -121,26 +122,51 @@ local function send_report(signal_type, t, host, port)
     end
   end
 
-  local sock = udp_sock()
-  local ok, err = sock:setpeername(host, port)
-  if not ok then
-    log(WARN, "could not set peer name for UDP socket: ", err)
-    return
-  end
+  local sock
+  if use_udp then
```

Comment:
Forgive if I'm missing the obvious, but is there any reason the UDP mode is still being kept around?

Reply:
@dndx, if someone decides later on to send udp, perhaps on some reports. We can name it protocol too, if someone later on wants us to send it with tls or perhaps grpc.

Reply:
@dndx I discussed this with @hishamhm, I'll remove the udp code.

Reply:
@dndx the udp is now removed.